uqcsbot/minecraft.py

```diff
@@ -48,6 +49,27 @@ async def send_rcon_command(self, command: str):
 
         return response
 
+    @app_commands.command()
+    async def mcplayers(self, interaction: discord.Interaction):
+        """Returns the number and list of people currently playing on the Minecraft server."""
+        server = JavaServer.lookup(
+            "minecraft.uqcs.org:25605"
+        )  # Does this need to be hard coded?? Is RCON addr/IP the same?
+        status = server.status()  # type: ignore
```

Comment:
Note for posterity. I'm conditionally approving this type:ignore, because it's part of a library. Will be following up and checking why, but that doesn't have to block this PR - I think it's to do with `mcserver` using Any type instead of actual object types.